### PR TITLE
Sbaelidev 196 about modal

### PR DESCRIPTION
**Pull request by clarityscalder**
To test run the front end and click Begin Your Journey.  There are four icons at the bottom of the dialog to allow jumping between pages.  The first three pages have a skip button.  The fourth page does not display the skip button.  Page 1 and Page 2 have a Next button.  Page 3 has a Register and Skip this Step button.  Page 4 has a Start First Learning Path button.

### CODE REVIEW

**Comment by clarityscalder** [comment]
1.  This is sort of fixed.  It's pretty buggy.  It will work more reliably with redux.
2.  Done.
3.  Done.
4.  I could not reproduce any issues with the little circles.  They seemed to work pretty reliably to me.  Any more guidance on how to reproduce the problem?

File: front-end/src/components/LearningPath/LearningPathAbout.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          <Grid verticalAlign='middle'>
+            <Grid.Row>
+              <Grid.Column width={4} textAlign="left">
+                { this.state.page > 1 ? <Button onClick={() => this.turnPage(-1)}>Back</Button> : "" }
+              </Grid.Column>
+              
+              <Grid.Column width={8} textAlign="center">
+                <Icon name="circle" onClick={() => this.setPage(1)} style={{marginRight: '10px', color: this.state.page === 1 ? 'green' : 'gray'}} />
+                <Icon name="circle" onClick={() => this.setPage(2)} style={{marginRight: '10px', color: this.state.page === 2 ? 'green' : 'gray'}} />
+                <Icon name="circle" onClick={() => this.setPage(3)} style={{marginRight: '10px', color: this.state.page === 3 ? 'green' : 'gray'}} />
```
Comment (arwagner) [inline front-end/src/components/LearningPath/LearningPathAbout.js]: Can we move these styles into a class in CSS please?

File: front-end/src/components/LearningPath/LearningPaths.js [hunk 2]
```diff
@@ -1,15 +1,40 @@
...
 
 class LearningPaths extends React.Component {
-  state  = {}
+  state  = {
+    displayModal: false
+  }
+  
+  componentDidMount() {
+    if(this.props.location.pathname === "/learning_paths" && this.props.location.state && this.props.location.state.showModal) {
+      this.setState({displayModal: true})
```
Comment (arwagner) [inline front-end/src/components/LearningPath/LearningPaths.js]: Do we need to check all 3 of these conditions? I think showModal is the only one we need to check, maybe?

Comment (clarityscalder) [reply front-end/src/components/LearningPath/LearningPaths.js]: This code will go away with Redux

File: front-end/src/components/LearningPath/LearningPathAbout.js (hunk 1 above)
Comment (clarityscalder) [reply front-end/src/components/LearningPath/LearningPathAbout.js]: Done.

**Pull request merged**